glotaran/project/project_parameter_registry.py

```diff
@@ -0,0 +1,116 @@
+"""The glotaran parameter registry module."""
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Literal
+
+from yaml import dump
```

Comment:
`yaml` isn't a builtin module and we changed for `pyyaml` to `ruamel.yaml`  use this instead so the benchmarks won't crash. This only works in the other tests because we get `pyyaml` as an indirect dependency from one of our test dependencies.